js/script.js

```diff
@@ -63,18 +60,53 @@
 		}
 	});
 
+	$(window).load(function(){
+		resizeSystemCharts();
+	});
+
 	$(window).resize(function () {
 		resizeSystemCharts();
 	});
 
-	function updateCPUStatistics (cpuload) {
+	/**
+	 * Reset all canvas widths on window resize so canvas is responsive
+	 */
+	function resizeSystemCharts() {
+		var cpu_canvas = $("#cpuloadcanvas"),
+			cpu_canvas_width = cpu_canvas.parents('.infobox').width() - 30,
+			mem_canvas = $("#memorycanvas"),
+			mem_canvas_width = mem_canvas.parents('.infobox').width() - 30,
+			active_users_canvas = $("#activeuserscanvas"),
+			active_users_canvas_width = active_users_canvas.parents('.infobox').width() - 30,
+			share_canvas = $("#sharecanvas"),
+			share_canvas_width = share_canvas.parents('.infobox').width() - 30,
+			current_vw = $(window).width();
+
+		// We have to set css width AND attribute width
+		cpu_canvas.width(cpu_canvas_width);
+		cpu_canvas.attr('width', cpu_canvas_width);
+		mem_canvas.width(mem_canvas_width);
+		mem_canvas.attr('width', mem_canvas_width);
+		active_users_canvas.width(active_users_canvas_width);
+		active_users_canvas.attr('width', active_users_canvas_width);
+		share_canvas.width(share_canvas_width);
+		share_canvas.attr('width', share_canvas_width);
+
+		updateShareStatistics();
+		updateActiveUsersStatistics();
+	}
+
+	function updateCPUStatistics(cpuload) {
+		var $cpu_footer_info = $('#cpuFooterInfo');
+		var $cpu_load_canvas = $('#cpuloadcanvas');
+
 		if (cpuload === 'N/A') {
-			$('#cpuFooterInfo').text(t('serverinfo', 'CPU info not available'));
-			$('#cpuloadcanvas').addClass('hidden');
+			$cpu_footer_info.text(t('serverinfo', 'CPU info not available'));
```

Comment:
Please use camelCase for variable for new code (we also have eslint rules but don't enforce them because lot of old code has to be migrated).

Reply:
i didn't notice that in the already present JS code. But know i have converted all variables to camelCase.